test/conformance/testdata/conformance.txt

```diff
@@ -191,6 +191,7 @@ test/e2e/network/service.go: "should serve multiport endpoints from pods"
 test/e2e/network/service_latency.go: "should not be very high"
 test/e2e/node/events.go: "should be sent by kubelets and the scheduler about pods scheduling and running"
 test/e2e/node/pods.go: "should be submitted and removed"
+test/e2e/node/pods.go: "should be submitted and removed"
```

Comment:
why do we have dupe entries here?

Reply:
two test cases are using same name. I will update it

Reply:
@spiffxp raised a PR #75544